extensions/metadata.md

```diff
@@ -722,7 +726,7 @@ As servers may rewrite values set by clients with `METADATA SET`, clients should
 
 This specification replaces the first `metadata` specification, by adding the following incompatible changes:
```

Comment:
```suggestion
This specification replaces the `metadata-notify` cap defined in the deprecated [metadata-3.2](https://ircv3.net/specs/core/metadata-3.2) specification, by adding the following incompatible changes:
```

The heading of this section should also be updated, but it's not in this changeset so github won't let me add a suggestion.

Reply:
the link won't exist next time we update the website

Reply:
```suggestion
This specification replaces an earlier deprecated `metadata-notify` specification, by adding the following incompatible changes:
```

Reply:
@progval happy with revised suggestion added above?